src/contents/articles/hello-world/index.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Hello World
+title: Introducing Namespace
 author: Marcus Sanatan
 date: 2015-05-24
 template: article.jade
 ---
 
-### This is **namespace**
-
 Hi everyone, we're officially launching namespace - a club for software developers in Trinidad and Tobago. We're a movement to remove the silos of people scattered throughout the nation that prevents us from build a community. 
```

Comment:
I like the idea that Namespace could be a movement in addition to be a club - gives us that upcoming, action-oriented feel. I'll admit that it probably isn't best to define it as a movement right after defining it as a club, some more context should be given.